1.3 - first program

In [ ]:
print("Hello, World!")

In [ ]:
40+2


In [ ]:
type(2)

In [ ]:
type(42.0)

In [ ]:
2++2


In [ ]:
2--2

In [ ]:
2+-2

In [ ]:
01+02

How mamy km in 10 miles?

In [4]:
km = 10
kmPerMil = 1.61
print( km / kmPerMil)

6.211180124223602


How many seconds in 42 mins and 42 seconds

In [2]:
print(((42*60)/1) + 42)

2562.0


average pace 
6.21 miles in 2562.0 seconds
pace = seconds * 1 / total miles

In [13]:
import datetime

km = 10
kmPerMile = 1.61
miles = km/kmPerMile
print("there are ",  miles, " miles in 10km")

pace = (2562 * 1) / miles
print("1 mile takes ", datetime.timedelta(seconds=pace), " seconds")



there are  6.211180124223602  miles in 10km
1 mile takes  0:06:52.482000  seconds
